subsys/bluetooth/controller/ll_sw/ull_conn_internal.h

```diff
@@ -81,3 +81,4 @@ void *ull_conn_ack_dequeue(void);
 struct ll_conn *ull_conn_tx_ack(u16_t handle, memq_link_t *link,
 				struct node_tx *tx);
 u8_t ull_conn_llcp_req(void *conn);
+void ull_conn_upd_curr_reset(void);
```

Comment:
Will this function be called from other modules? If not it should be static.

Reply:
This is called from ull_sched.c